vault/external_tests/mfa/login_mfa_test.go

```diff
@@ -195,6 +236,13 @@ func TestLoginMFA_Method_CRUD(t *testing.T) {
 				t.Fatal("expected response id to match existing method id but it didn't")
 			}
 
+			// update the name with an invalid namespace prefix should fail
```

Comment:
Why is this something we do for each test case, instead of it being its own test case?

Reply:
In general it feels like most (all?) of the changes you made to this method would be clearer if instead they appeared in a new method and/or new test cases of this method.